docs/DeveloperGuide.md

```diff
@@ -69,7 +69,7 @@ If you plan to use IntelliJ IDEA (highly recommended):
     No setup is required for this.
     
 1. Learn the design
-    * When you are ready to start coding, we recommend that you look at [ModTracker's architecture](#architecture) to get some 
+    * When you are ready to start coding, we recommend that you look at [ModTracker's architecture](#3.1) to get some 
     sense of the overall design.
 
 
```

Comment:
Is there something missing in this diagram?

![Screenshot 2020-10-29 142024](https://user-images.githubusercontent.com/48957360/97532743-f13c3680-19f1-11eb-9b69-b552af68f3bc.png)